addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/model/OshiSysteminfo.java

```diff
@@ -393,6 +427,20 @@ public DecimalType getSwapAvailablePercent() {
         }
     }
 
+    @Override
+    public DecimalType getSwapUsedPercent() {
+        long usedSwap = memory.getSwapUsed();
+        long totalSwap = memory.getSwapTotal();
+        BigDecimal usedPercent;
```

Comment:
To be honest I just copy&paste the `getSwapAvailablePercent` method and changed the logic. ;-)

But both annotation make sense. If I change it in this place, I should refactor the remaining methods too.